environments/staging/terraform.yml

```diff
@@ -219,30 +219,6 @@ servers:
     group: "couchdb2"
     os: ubuntu_pro_bionic
 
-  - server_name: "couch10-staging"
-    server_instance_type: "t3.large"
-    network_tier: "db-private"
-    az: "a"
-    volume_size: 30
-    volume_encrypted: yes
-    block_device:
-      volume_size: 80
-      encrypted: true
-    group: "couchdb2"
-    os: ubuntu_pro_bionic
-
-  - server_name: "couch11-staging"
-    server_instance_type: "t3.large"
-    network_tier: "db-private"
-    az: "a"
-    volume_size: 30
-    volume_encrypted: yes
-    block_device:
-      volume_size: 80
-      encrypted: true
-    group: "couchdb2"
-    os: ubuntu_pro_bionic
-
```

Comment:
This change (removing these two servers) doesn't seem to fit into this PR. Were these machines already terminated and this is just catching up to that? Either way, would be good to have as a separate PR

Reply:
yes, Danny. These instances are terminated and PR was also merged into master.